makefile

```diff
@@ -77,16 +76,16 @@ package-rpm:
 .PHONY: test
 test:
 	@echo "Testing daemon"
-	go test -cover ${DAEMON_ONLY_PKGS}
+	go test -cover ./...
 
 vet:
-	${GO_VET_CMD} ${SDK_BASE_FOLDERS}
+	${GO_VET_CMD} ./...
```

Comment:
NP: Consider eliminating `GO_VET_CMD` and replacing with `go vet` to be consistent with other Makefile targets. According to the [docs](https://godoc.org/github.com/golangci/govet#hdr-Other_flags), the `-all` flag is unnecessary.

Reply:
Good catch!